harpoon/commands/gn.py

```diff
@@ -16,16 +15,19 @@ class CommandGreyNoise(Command):
     """
     # GreyNoise
 
-    See https://github.com/Grey-Noise-Intelligence/api.greynoise.io
+    See https://developer.greynoise.io/
+
+    * To use GreyNoise Community (Free) API, set api_type = "community" in config.
+    * Commuinty API only supports IP lookup command
 
     * List tags: `harpoon greynoise -l` (default output as json)
     * Search for an IP: `harpoon greynoise -i IP`
     * Run a GNQL query: `harpoon greynoise -q "classification:malicious tags:'emotet'"`
     """
 
     name = "greynoise"
-    description = "Request Grey Noise API"
-    config = {"GreyNoise": ["key"]}
+    description = "Request information from GreyNoise API"
+    config = {"GreyNoise": ["key", "api_type"]}
```

Comment:
Added a new input to specify if harpoon should use the GN Paid or Community API.